package.json

```diff
@@ -49,7 +49,8 @@
     "prettier-standard": "^6.0.0",
     "rimraf": "^2.6.1",
     "snazzy": "^7.0.0",
-    "standard": "^10.0.2"
+    "standard": "^10.0.2",
+    "tachyons": "^4.8.0"
```

Comment:
Tachyons is not dev dependency
You should install it with the module manager (npm or yarn). This will add the dep automatically ('npm i tachyons --save` or 'yarn add tachyons`) to package json and also update the lockfile.

Reply:
- also remove unused dependencies `suitcss-base` and `suitcss-utils`
- also use a  package manager when removing